Cargo.toml

```diff
@@ -16,6 +16,8 @@ license = "MIT"
 [dependencies]
 rand = { version = ">= 0.3.10, < 0.7", optional = true }
 serde = { version = "1.0", features = ["derive"], optional = true}
+digest = { version = "0.8", default-features = false }
```

Comment:
These will need to be made optional dependencies to allow people who don’t need them to avoid them.

Reply:
Fixed :)